packages/server/lib/socket-base.ts

```diff
@@ -338,13 +337,6 @@ export class SocketBase {
           return options.onMocha.apply(options, args)
         })
 
-        socket.on('open:finder', (p, cb = function () {}) => {
```

Comment:
I couldn't find this actually emitted anywhere. Maybe I'm wrong in my understanding of this though.